src/bls/bls_aggregator.cpp

```diff
@@ -32,8 +33,31 @@
 
 namespace eth {
 
+// When a service node receives a request to sign an exit request for a BLS node, this values
+// determines how old that request is allowed to be from the time it was initiated to us receiving
+// it, that we will consider signing it.
+inline constexpr auto BLS_MAX_TIME_ALLOWED_FOR_EXIT_REQUEST = 3min;
```

Comment:
```suggestion
constexpr auto BLS_MAX_TIME_ALLOWED_FOR_EXIT_REQUEST = 3min;
```
`inline` was only there before because it was in the header.

Reply:
Yep, removed the inline.